docs/agents/sematext-agent/kubernetes/operator.md

```diff
@@ -0,0 +1,73 @@
+# Sematext Operator
+
+The Sematext Operator for Kubernetes provides an easy way to deploy Sematext Agent.
+
+
+It installs the Sematext Agent to all nodes in your cluster via a `DaemonSet` resource.
+
+## Quickstart
+
+To run the operator and its dependencies run the following command:
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/sematext/sematext-operator/master/bundle.yaml
```

Comment:
I see several versions in that yaml.  Are they all up to date?

Reply:
Will wait for this before approving the PR

Reply:
They are not. SC-13312 should fix that, since to update our presence on OperatorHub we will publish a new Sematext Operator based on our latest Helm chart, which is up to date